src/components/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -0,0 +1,54 @@
+import { SxProps, Theme } from '@mui/material/styles';
+import { ReactNode } from 'react';
+import { HorizontalTab } from './HorizontalTab';
+import {
+  HorizontalTabsContainer,
+  HorizontalTabSize,
+} from './HorizontalTabs.style';
+export interface HorizontalTabItem {
+  value: string;
+  label?: ReactNode;
+  startAdornment?: ReactNode;
+  endAdornment?: ReactNode;
+  disabled?: boolean;
+}
+
+export interface HorizontalTabsProps {
+  tabs: HorizontalTabItem[];
+  onChange: (event: React.SyntheticEvent, newValue: string) => void;
+  onTabClick: (
+    value: string,
+  ) => (event: React.MouseEvent<HTMLDivElement>) => void;
+  value?: string;
+  size?: HorizontalTabSize; //@todo: use enum instead
+  sx?: SxProps<Theme>;
+  tabSx?: SxProps<Theme>;
+}
+
+export const HorizontalTabs = ({
+  tabs,
+  onChange,
+  onTabClick,
+  value,
+  size = HorizontalTabSize.LG,
+  sx,
+  tabSx,
```

Comment:
Seems overkill to pass those through sx, is that really needed?

Reply:
Yes, it is making things much harder to read + subject to errors/regression (this is the reason why we removed most of those implementation a year ago).

Benefits seems to be very limited as the tab itself should scale on the tabsContainer, if tab itself require different styling, we can edit it from the container sx (with class props) 

Especially as it is not even used :/